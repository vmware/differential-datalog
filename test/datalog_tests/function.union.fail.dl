// Union types must be specified by name

function(x : Constructor{}): int
