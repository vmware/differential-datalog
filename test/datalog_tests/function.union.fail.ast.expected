Failed to parse input file: "./test/datalog_tests/function.union.fail.dl" (line 3, column 9):
unexpected "("
expecting end of "function" or identifier